packages/experiments/src/components/Button/Button.types.tsx

```diff
@@ -73,219 +53,23 @@ export type INativeButtonProps = Omit<React.AllHTMLAttributes<HTMLAnchorElement
  */
 export interface IButtonProps
   extends ISlottableProps<IButtonSlots>,
+    Pick<
+      IBaseButtonProps,
+      'href' | 'primary' | 'circular' | 'disabled' | 'checked' | 'allowDisabledFocus' | 'ariaLabel' | 'keytipProps' | 'uniqueId'
+    >,
     IStyleableComponentProps<IButtonProps, IButtonTokens, IButtonStyles>,
     IBaseProps<IButton>,
-    INativeButtonProps {
-  /**
-   * Defines an href reference that, if provided, will make this component render as an anchor.
-   */
-  href?: string;
-
-  /**
-   * Defines whether the visual representation of the Button should be emphasized.
-   * @defaultvalue false
-   */
-  primary?: boolean;
-
-  /**
-   * Defines whether the Button should be circular.
-   * In general, circular Buttons should not specify the menu and container slots.
-   * @defaultvalue false
-   */
-  circular?: boolean;
-
-  /**
-   * Defines whether the Button is disabled.
-   * @defaultvalue false
-   */
-  disabled?: boolean;
-
-  /**
-   * Defines whether the Button is in a checked state (for toggle buttons).
-   */
-  checked?: boolean;
-
-  /**
-   * Defines whether disabled buttons should be tabbable via keyboard navigation or not.
-   * @defaultvalue false
-   */
-  allowDisabledFocus?: boolean;
-
-  /**
-   * Defines the aria label that the screen readers use when focus goes on the Button.
-   */
-  ariaLabel?: string;
-
-  /**
-   * Defines optional keytips for this button.
-   */
-  keytipProps?: IKeytipProps;
-
-  /**
-   * Defines an unique id to identify the item. Typically a duplicate of key value.
-   */
-  uniqueId?: string | number;
-}
+    INativeButtonProps {}
 
 /**
  * {@docCategory Button}
  */
-export interface IButtonViewProps extends IButtonProps {
-  /**
-   * Defines a reference to the inner Button.
-   */
-  buttonRef?: React.RefObject<HTMLButtonElement>;
-}
+export interface IButtonViewProps extends Pick<IBaseButtonViewProps, 'buttonRef'>, IButtonProps {}
```

Comment:
Hmm, we don't want to expose `buttonRef` in `IButtonProps`?

Reply:
See above.